appveyor.yml

```diff
@@ -32,12 +32,19 @@ test_script:
 build_script:
 - make cross
 
+after_build:
```

Comment:
Wouldn't it be better to run this after tests (`after_tests`)?
Currently, it builds binaries, than gzips it, and then run tests. But gzipping doesn't make sense if tests don't pass.

Reply:
yep, ack!